net/raft/raft_test.go

```diff
@@ -116,3 +119,115 @@ func TestStartUninitialized(t *testing.T) {
 		t.Errorf("sv.Exec() = %s, want %s", err, ErrUninitialized)
 	}
 }
+
+func TestClusterSetup(t *testing.T) {
+	ctx := context.Background()
+
+	// Create three uninitialized raft services.
+	nodeA := newTestNode(t)
+	defer nodeA.cleanup()
+	nodeB := newTestNode(t)
+	defer nodeB.cleanup()
+	nodeC := newTestNode(t)
+	defer nodeC.cleanup()
+
+	// Initialize A, creating a fresh cluster.
+	must(t, nodeA.service.Init())
+
+	// Update the cluster to allow A, B and C's addresses.
+	var err error
+	_, err = nodeA.service.Exec(ctx, set("/allowed/"+nodeA.addr, "yes"))
```

Comment:
¯\\\_(ツ)\_/¯, it's for the test state implementation, and it uses the presence of the key not the value to determine if an address is allowed. So it can be anything.